<launch>
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/odroid/image_percept/down_image/output_video"/>
        <param name="autosize" value="true" /> 
    </node>
</launch>
